src/localization/en.json

```diff
@@ -140,7 +140,9 @@
     "vacationType": "Vacation Type:",
     "applicant": "Applicant:",
     "timeOfVacation": "Time of Vacation:",
-    "status": "Status:"
+    "status": "Status:",
+    "spentVacations": "Spent vacation days",
+    "unspentVacations": "Unspent vacation days"
```

Comment:
You use colons : for the finnish translation so lets do the same for the english. Also use of a space after the colon is fine rather than the styling option as mentioned in a previous comment. Either way is fine but lets be consistent

Reply:
Silly mistake, supposed to have a colon + space. Fixed now.